site/gatsby-site/cypress/e2e/integration/navigation.cy.js

```diff
@@ -1,37 +1,37 @@
-describe.skip('Navigation', { retries: { runMode: 4 } }, () => {
-  it('Check menu links work (English)', () => {
-    cy.visit('/');
+it('Check menu links work (English)', () => {
```

Comment:
I wonder what the consequences of missing a `describe` call are, but I would add it for the sake of consistency.